index/index.js

```diff
@@ -24,85 +24,193 @@ function weekDiff(start, end) {
 }
 
 // Calls the banner proxy and returns class info in JSON format
-const invokeLambda = async(params) => {
-  const data = await lambda.invoke(params).promise();
-  data.Payload = data.Payload.replace(/\+/g, '');
-  return JSON.parse(data.Payload);
+async function invokeLambda(params) {
+  try {
+    const data = await cache.lambda.invoke(params).promise();
+    data.Payload = data.Payload.replace(/\+/g, '');
+    return JSON.parse(data.Payload);
+  } catch (err) {
+      console.log(err);
+      return err;
+  }
 }
 
-exports.handler = async(event, context) => {
-  const db = new AWS.DynamoDB.DocumentClient({ region: "us-east-2" });
-  
+// Calls the banner proxy and returns campues names & codes
+async function invokeCampusLambda(params) {
+  try {
+    const data = await cache.campusLambda.invoke(params).promise();
+    data.Payload = data.Payload.replace(/\+/g, '');
+    return JSON.parse(data.Payload);
+  } catch (err) {
+      console.log(err);
+      return err;
+  }
+}
+
+// Puts items into database 
+async function putIntoDB(items) {
+  cache.db.batchWrite(items, function(err, data) {
+    if(err) {
+      console.log(err);
+    } else {
+      console.log(data);
+    }
+  }); 
+}
```

Comment:
Don't mix callbacks and promises. Use one or the other (Promises are preferred and the more modern way of doing asynchronous operations).